src/components/sidebar/Sidebar.jsx

```diff
@@ -14,12 +20,34 @@ export const Sidebar = () => {
             className={({ isActive }) => (isActive ? "active-link" : "")}
           >
             <li>
-              <span className="sidebar-icon">{item.icon}</span>
+              <span>{item.icon}</span>
               {item.linkName}
             </li>
           </NavLink>
         ))}
       </ul>
+      <ul className="personal_links">
+        <li>
+          <a href="https://github.com/vleads" target="_blank">
+            <GithubIcon />
+          </a>
+        </li>
+        <li>
+          <a href="linkedin.com/in/vishalkumar28/" target="_blank">
+            <LinkedinIcon />
+          </a>
+        </li>
+        <li>
+          <a href="https://twitter.com/vishalk01234" target="_blank">
+            <TwitterIcon />
+          </a>
+        </li>
+        <li>
+          <a href="mailto:leader.vishalkumar@gmail.com" target="_blank">
+            <MailIcon />
+          </a>
+        </li>
```

Comment:
most of the jsx here is repeating, can make an array of object and loop over them ??

Reply:
This is being only used at one place and for now they are only 4. So, I have implemented them directly on the very page without data.